ui/dashboardApp/prefix.ts

```diff
@@ -0,0 +1,9 @@
+let dashboardPrefix =
+  document
+    .querySelector('meta[name=x-dashboard-prefix]')
+    ?.getAttribute('content') || '/dashboard'
+if (dashboardPrefix === '__DASHBOARD_PREFIX__') {
+  dashboardPrefix = '/dashboard'
+}
+
+export { dashboardPrefix }
```

Comment:
prefer not to repeat `dashboard` in dashboard.. A better name could be `urlPrefix` or `publicPathPrefix`